.github/workflows/pr_push_workflow.yml

```diff
@@ -0,0 +1,62 @@
+name: PR/push
+
+on: [push, pull_request]
+
+concurrency: 
+    group: ${{ github.head_ref }}-${{ github.workflow }}
+    cancel-in-progress: true
+
+jobs:
+    Fast_build:
```

Comment:
Every name is SpelledTogetherLikeThis and this_one_is_not 😉

Reply:
Done